src/SFA.DAS.Authorization.EmployerFeatures/Handlers/AuthorizationHandler.cs

```diff
@@ -48,6 +52,17 @@ public Task<AuthorizationResult> GetAuthorizationResult(IReadOnlyCollection<stri
                         authorizationResult.AddError(new EmployerFeatureUserNotWhitelisted());
                     }
                 }
+                if (
```

Comment:
Minor point but this probably doesn't need to be run if `featureToggle.IsEnabled` is `false`.